docs/io.cozy.identities.md

```diff
@@ -37,22 +37,57 @@ unique by identifier **and** cozyMetadata.createdByApp
         "familyName": "Dupond",
         "givenName": "Jean"
       },
-      "phone": [
+      "phones": [
         {
           "number": "06 06 06 06 06",
           "primary": true,
           "type": "mobile"
         }
-      ]
+      ],
+      "maritalStatus": "married",
+      "numberOfDependants": 1
   },
+  "tax_informations": [
+    {
+      "year": "2022",
+      "RFR": "56789.10",
+      "1AJ": "12345.67",
+      "1BJ": "23456.78",
+      "currency": "EUR",
+      "file": "avis_impot_2022.pdf"      
```

Comment:
Not sure that the filename is useful since it can be renamed for instance. Can't we use a `relationship` here? 

```
"tax_informations": [
    {
      "year": "2022",
      "RFR": "56789.10",
      "1AJ": "12345.67",
      "1BJ": "23456.78",
      "currency": "EUR",
      "relationship": {
           "type": "io.cozy.files", 
           "_id": XXX
       }
```